src/viewer/README.md

```diff
@@ -0,0 +1,11 @@
+## Viewer Component
+
+### TODO
+
+These components should be standalone, but they currently have dependencies on a Cordova plugin and some Cordova helper functions. To remedy that, the plan is to:
+
+- Provide only a set of basic, truly standalone viewers here
+- Add the ability to configure the module to use specific Viewers under certain conditions
+- Provide the `NativePdfViewer` through these options, as a replacement for the default PDF viewer in native apps
+- Reduce the `NoViewer` to a more simple version of itself
+- Provide an extended `NoViewer` that can open files in native apps
```

Comment:
Could you add checkbox for each item. This way we could check advancement when needed. In markdown, just add brackets : 

```md
- [ ] uncheked
- [x] checked
```

- [ ] uncheked
- [x] checked

Reply:
I think the list could be less detailed:

- [ ] remove third-party dependence
- [ ] add options and configuration to inject dependance and other fallbacks
- [ ] the whole viewer folder do not depends on anything other than external libs (for instance: cozy-client or cozy-ui)

Reply:
I think detailed is good. It's not like the bullet points are very long either.